php/libraries/Database.class.inc

```diff
@@ -271,7 +271,11 @@ class Database
     {
         $retVal = array();
         foreach ($set as $key => $val) {
-            $retVal[$key] = is_null($val) ? null : htmlspecialchars($val);
+            if ($val === 'Unrated') {
+                $retVal[$key] = '';
+            } else {
+                $retVal[$key] = is_null($val) ? null : htmlspecialchars($val);
```

Comment:
isn't this the opposite of the null coalesce operator?

(In fact, I don't see why this justifies a ternary operator at all, when null can just be handled as another `else if` case in the `if` that already exists..)